db/migrate/20160324174132_create_container_deployments.rb

```diff
@@ -0,0 +1,22 @@
+class CreateContainerDeployments < ActiveRecord::Migration[5.0]
+  def change
+    create_table :container_deployments do |t|
+      t.string     :deployment_type
+      t.string     :deployment_version
+      t.boolean    :containerized
+      t.string     :deployment_method
+      t.string     :rhsm_user
+      t.string     :rhsm_pass
+      t.string     :rhsm_sku
+      t.string     :rhsm_pool_id
+      t.string     :rhsm_server
+      t.string     :rhsm_activation_key
```

Comment:
That is, a container_deployment could have an authentication record of type AuthUseridPassword or perhaps a new type of AuthenticationRedHatSubscriptionManager with an authtype of `"rhsm"`